scripts/tests/chiptest/test_definition.py

```diff
@@ -239,16 +240,18 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str):
             # so it will be commissionable again.
             app.factoryReset()
             app.start(str(randrange(1, 4096)))
-
-            runner.RunSubprocess(
-                tool_cmd + ['pairing', 'qrcode', TEST_NODE_ID, app.setupCode] +
-                ['--paa-trust-store-path', DEVELOPMENT_PAA_LIST],
-                name='PAIR', dependencies=[apps_register])
-
+            trust_store_path = [] if sys.platform == 'darwin' else [
```

Comment:
Previously code always had ppa store. Why did this become platform-dependent? PR summary does not seem to provide details.

Postfix if is hard to read and so is multiline addition with '\'. Can you refactor the code to feel more python-like?

```py
pairing_cmd = tool_cmd +['pairing', 'qrcode', TEST_NODE_ID, app.setupCode]

if sys.platform == 'darwin':
   pairing_cmd.append('--paa-trust-store-path')
   pairing_cmd.append(DEVELOPMENT_PAA_LIST)

```

and then fixing_cmt too. It is ok to duplicate some code if the result is more readable.

Reply:
I will update this on the new PR. Thank you.